monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -80,6 +80,15 @@ def __init__(self, template_path: str):
         # track records when filling template config: {"<config name>": {"<placeholder key>": value, ...}, ...}
         self.fill_records: dict = {}
 
+    def pre_check_skip_algo(self):
```

Comment:
Maybe we can make the function inputs more general? For example `pre_check_skip_algo(self, skip_bundlegen: bool = False, skip_info:str = "")

Reply:
I recommend this function to return a tuple (bool, str), E.g.
return (False, '') - default, and similar from each "algo" that supports it

In this case, you don't need to set class variables self.skip_bundlegen, and self.skip_info
since the only place that information is used is in generate() function

Reply:
@myron Thanks Andriy, I've changed the function to return a tuple.